src/activities/Elsa.Activities.Mqtt/Services/Worker.cs

```diff
@@ -30,6 +30,11 @@ public class Worker
             _receiverClient.SetMessageHandlerAsync(OnMessageReceived);
         }
 
+        public void Ping()
+        {
+            _receiverClient.SetMessageHandlerAsync(OnMessageReceived);
+        }
+
```

Comment:
Out of curiosity, can you explain how this works? The method is called "ping", but it only sets a message handler. Does this mean that the receiver client is going to send some "ping" message under the hood?

Reply:
Yeah i had to adopt a compromise, because i have to leave opened subscribing (before i have issue with retain topics)  so instead to open and close, now i test if  Client.IsConnected